ctk/common/src/main/kotlin/org/codice/compliance/verification/core/assertions/NameIdentifierVerifier.kt

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.compliance.verification.core.assertions
+
+import org.codice.compliance.allChildren
+import org.codice.compliance.verification.core.CommonDataTypeVerifier
+import org.w3c.dom.Node
+
+internal class NameIdentifierVerifier(val node: Node) {
+    companion object {
+
+        /** 2.2.2 Complex Type NameIDType */
+        fun verifyNameIDType(node: Node) {
```

Comment:
❗️ Missed this the first time around, but NameIdType can contain NameQualifier and SPNameQualifier attributes too.